### DISCUSSION TITLE
How to download images from galleries?

### DISCUSSION THREAD

**Discussion by mustafa-git** [Docile-Alligator/Infinity-For-Reddit#1238] [Q&A] [answered]
The bar that has the options to download images, sharing them, or setting them as wallpapers is missing on gallery images as far as I know; and it doesn't show when you single tap like with single image posts.

**Comment by starchyunderscore** [comment]
![Screenshot_20221128-111355_Infinity.jpg](https://user-images.githubusercontent.com/102399926/204327106-1f41c065-9582-49ba-878d-d974ee7e29ee.jpg)

its there for me, latest relese.

**Reply by mustafa-git** [reply]
I guess it just doesn't show when you have the toolbar at the bottom. ![Screenshot_20221217_174233_ml.docilealligator.infinityforreddit.jpg](https://user-images.githubusercontent.com/89876073/208249882-9f278a25-c530-4dd3-b885-67910719dd07.jpg)![Screenshot_20221217_174226_ml.docilealligator.infinityforreddit.jpg](https://user-images.githubusercontent.com/89876073/208249889-deb0e393-bc45-4aff-a121-4b9168d0a43e.jpg)![Screenshot_20221217_174118_ml.docilealligator.infinityforreddit.jpg](https://user-images.githubusercontent.com/89876073/208249893-c1b92cb9-d747-4698-9500-5eda5f284c5e.jpg)![Screenshot_20221217_174113_ml.docilealligator.infinityforreddit.jpg](https://user-images.githubusercontent.com/89876073/208249896-9bddfc68-4de1-4d6e-9421-5360e6e02021.jpg)

**Comment by Docile-Alligator** [comment, marked as answer]
It's fixed in v5.4.0-beta1.
